game/CMakeLists.txt

```diff
@@ -303,4 +311,38 @@ if(WIN32)
             message(WARNING "Runtime dependency ${dep} could not be resolved.")
         endforeach()
         ]])
+elseif(MINGW)
+	install(
+		CODE [[
+		if(DEFINED ENV{ORIGINAL_TEMP})
+			set (TMP_LOCATION $ENV{ORIGINAL_TEMP})
+		elseif(DEFINED ENV{TEMP})
+			set (TMP_LOCATION $ENV{TEMP})
+		else()
+			message(WARNING "Temp folder not found. CPack won't be able to create a proper installer.")
+		endif()
+		file (READ "${TMP_LOCATION}/source.txt" CPACK_TOP_SOURCE_DIR)
+		file (READ "${TMP_LOCATION}/mingwroot.txt" CPACK_MINGW_SYSTEM_PREFIX)
+		message ("SOURCE DIR IS: ${CPACK_TOP_SOURCE_DIR}")
+		message ("HOME DIR IS: ${CMAKE_HOME_DIRECTORY}")
+		message ("INSTALL DIR IS: ${CMAKE_INSTALL_PREFIX}")
```

Comment:
This should all be done automaticly. See previous cpack_install_dir/build_dir comment

Reply:
It is not done automatically and I swear I lost almost the entire weekend trying to get it to work. It's a fucking nightmare.